src/core/network/polling.ts

```diff
@@ -14,6 +14,7 @@ const noop = always(Promise.resolve())
 export class Polling {
   private readonly abortController = new AbortController()
   private offset = 0
+  public skipOffsetSync = false
```

Comment:
It is fine that this is `public` because it still cannot be reached from the outside, as `Polling` is never exposed.

It cannot be made `private` becuase `Member 'skipOffsetSync' is never reassigned; mark it as `readonly`. (eslint@typescript-eslint/prefer-readonly)` which makes sense, but making it `readonly` would defeat the point.